src/mlpack/tests/test_tools.hpp

```diff
@@ -48,4 +48,18 @@ inline void CheckMatrices(const arma::Mat<size_t>& a,
     BOOST_REQUIRE_EQUAL(a[i], b[i]);
 }
 
+inline std::string FilterFileName(const std::string& inputString)
```

Comment:
Can you add a comment to mention what this function is for please? Even in the tests we should be sure we have enough documentation. :)